src/EditorFeatures/Core/Shared/Utilities/IThreadingContext.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Threading;
+
+namespace Microsoft.CodeAnalysis.Editor.Shared.Utilities
+{
+    internal interface IThreadingContext
+    {
+        bool HasMainThread
+        {
+            get;
+        }
```

Comment:
tiny nit: I think we format interface properties as a single-line throughout the code base.

Reply:
➡️ Opted against this for readability, especially with the upcoming increase in documentation comments. Would reconsider if analyzer enforcement was involved, but today the closest related preference (`csharp_preserve_single_line_blocks`) isn't even listed in our **.editorconfig**.